src/CoderDojo/WebsiteBundle/Form/Type/CocRequestFormType.php

```diff
@@ -50,13 +50,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label' => 'Achternaam'
             ))
             ->add('email', EmailType::class, [
-                'label' => 'Email'
+                'label' => 'Email',
+                'attr' => [
+                    'placeholder' => 'Let op, dit moet een persoonlijk emailadres zijn.'
```

Comment:
What is the intention of this placeholder? Are people using the info@ email addresses?